src/lib/rule2A.ts

```diff
@@ -0,0 +1,56 @@
+import {Context} from './context';
+
+/**
+ * Looks at a variety of characteristics (CSS, size on screen, attributes)
+ * to determine if 'node' should be considered hidden
+ * @param node - node whose hidden-ness is being calculated
+ * @return - whether or not the node is considered hidden
+ */
+function isHidden(node: Node): boolean {
+  if (node instanceof HTMLElement) {
+    const nodeStyle = window.getComputedStyle(node);
+
+    const hiddenByStyle = (
+      nodeStyle.visibility === 'hidden' ||
+      // measurement of pixels used on screen (checks for display:none)
+      node.offsetHeight <= 0 ||
+      node.offsetWidth <= 0
```

Comment:
I think we want an `&&` between those two conditions to check for `display: none`